aws/convert_types.go

```diff
@@ -311,6 +311,15 @@ func TimeValue(v *time.Time) time.Time {
 	return time.Time{}
 }
 
+// Int64TimeValue converts an int64 pointer to a time.Time value or
+// time.Time{} if the pointer is nil.
+func Int64TimeValue(v *int64) time.Time {
```

Comment:
It would be helpful for this to specify the resolution the `int64`is expected to be. In this case it looks to be millisecond -> second. Is this correct?

Alternatively dropping the `Int64` part all together and just `SecondsTimeValue` and `MillisecondTimeValue` with int64 input param. Bit more verbose, but clear on the expectations.